hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -235,19 +237,20 @@ spec = around showLogsOnFailure $ do
                 bobNodeId = 2
                 allNodesIds = [aliceNodeId, bobNodeId]
                 withAliceNode = withHydraNode tracer aliceChainConfig tmp aliceNodeId aliceSk [bobVk] allNodesIds hydraScriptsTxId
+                withBobNode :: (HydraClient -> IO a) -> IO a
```

Comment:
If you add it here, also add it above. But I would suggest to remove this signature.

Reply:
I added it above also since it is a bit hard to convince GHC that these `with...` functions can return polymorphic results.